packages/core/src/common/logger.ts

```diff
@@ -232,8 +236,13 @@ export class Logger implements ILogger {
                 }
             });
         });
+       if (serverOptions !== undefined) {
+          this._serverOptions = serverOptions;
+          server.create(serverOptions);
```

Comment:
I am not sure how it is supposed to work from the client-side. I don't think that class should be changed.